stdlib/http/src/main/ballerina/http/http_response.bal

```diff
@@ -316,8 +316,10 @@ public function Response.setETag(json|xml|string|byte[] payload) {
 
 public function Response.setLastModified() {
     time:Time currentT = time:currentTime();
-    string lastModified = time:format(currentT, time:TIME_FORMAT_RFC_1123);
-    self.setHeader(LAST_MODIFIED, lastModified);
+    var lastModified = time:format(currentT, time:TIME_FORMAT_RFC_1123);
+    if (lastModified is string) {
+        self.setHeader(LAST_MODIFIED, lastModified);
+    }
```

Comment:
How the error scenario should be handled in this case?

Reply:
Practically it is unlikely an error occurred here since the time is a current time and format is a constant.  Theoretically if some error occurred last modified time header won't get set in the response. But since this error cannot be happened no need of handling it explicitly

Reply:
Yes, it's unlikely that this would result in an error. IIRC, the spec doesn't say anything about handling a case where the current time cannot be parsed.

Reply:
I wasn't talking about the exact context here.
I'd say, if we an error scenario is possible, but we most certainly know it doesn't happen depending on the context, it would be best if we have an assertion for the error case due to the following reasons.
1. Someone may later modify return values of the relevant function and won't handle all cases. 
2. If the implementation changes someday in a way that an error probability is higher for the same scenario, that will not go unnoticed easily.

I'm suggesting this considering how we treat default case in java switch in a similar scenario. 
I know the above can be avoided if whoever changes it does that the proper way.

Reply:
@Manuri Agreed to your point here. Added a panic for error case (even  though it is unlikely to happen)